processor/internal/transformer/destination_transformer/embedded/warehouse/uploader.go

```diff
@@ -101,11 +100,6 @@ func (t *Transformer) sampleDiff(events []types.TransformerEvent, legacyResponse
 		return "" // Don't diff in case there is no response from transformer
 	}
 
-	sortTransformerResponsesByJobID(legacyResponse.Events)
```

Comment:
Why were we sorting ?

Reply:
So as to compare the event. In rudder-tranformer it's sequentially. But in warehouse transformations, we are processing concurrently. 
So before comparing, we are sorting to make sure they are of the same order.

Reply:
@achettyiitr To clarify we are going ahead with safe approach of non concurrency than changing logic which avoids sorting?

Reply:
Yeah. If we need to avoid sorting, there is an extra complexity around either deep copying or during comparision.